webplugin/js/app/media/media-service.js

```diff
@@ -120,8 +126,14 @@ Kommunicate.mediaService = {
                         );
                     }
                 };
-                isChrome && speechSynthesis.cancel();
+                if (isChrome) {
```

Comment:
Do we need to add Opera and Edge check?

Reply:
This check will work on Opea and Edge as well. 
Verifed that by testing in the respective browser.